src/test/java/edu/hawaii/its/api/service/TestMemberAttributeService.java

```diff
@@ -204,40 +138,80 @@ public void memberAttributesTest() {
     }
 
     @Test
-    public void membersAttributesTest() {
-        List<Subject> subjects = memberAttributeService.getMembersAttributes(ADMIN, testUids);
-        assertNotNull(subjects);
+    public void memberAttributeResultsTest() {
+        MemberAttributeResults results = memberAttributeService.getMemberAttributeResults(ADMIN, testUids);
+        assertNotNull(results);
+        HashSet<String> testUidsSet = new HashSet(testUids);
+        HashSet<String> testUhUuidsSet = new HashSet(testUhUuids);
+
+        for (MemberResult memberResult : results.getResults()) {
+            assertTrue(testUidsSet.contains(memberResult.getUid()));
+            assertTrue(testUhUuidsSet.contains(memberResult.getUhUuid()));
+        }
+
+        String testUid = testUids.get(0);
+        List<String> testList = new ArrayList<>();
+        testList.add(testUid);
+
+        // Should not contain memberResult of invalid uhIdentifier.
+        List<String> uhIdentifiers = new ArrayList<>();
+        uhIdentifiers.add("bogus-user");
+        results = memberAttributeService.getMemberAttributeResults(ADMIN, uhIdentifiers);
+        assertNull(results.getResults());
```

Comment:
For this test, I you should be checking `assertEquals(new ArrayList<>(), results.getResults())` and `assertEquals(uhIdentifiers, results.getInvalid())`.